notifier/management/commands/startbot.py

```diff
@@ -0,0 +1,11 @@
+from django.core.management.base import BaseCommand;
+from ...bot.botController import BotController;
```

Comment:
try not do far relative import, I see u came from React.js
instead do, `from notifier.bot.botController import BotController`
and follow the PEP convention